src/packaging/github/kernel/README.md

```diff
@@ -5,47 +5,51 @@ compressed block storage.
 
 ## Background
 
-VDO (which includes [kvdo](https://github.com/dm-vdo/kvdo) and
-[vdo](https://github.com/dm-vdo/vdo)) is software that provides inline
-block-level deduplication, compression, and thin provisioning capabilities for
-primary storage. VDO installs within the Linux device mapper framework, where
-it takes ownership of existing physical block devices and remaps these to new,
-higher-level block devices with data-efficiency capabilities.
+VDO is a device-mapper target that provides inline block-level deduplication,
+compression, and thin provisioning capabilities for primary storage. VDO
+is managed through LVM and can be integrated into any existing storage stack.
 
 Deduplication is a technique for reducing the consumption of storage resources
 by eliminating multiple copies of duplicate blocks. Compression takes the
 individual unique blocks and shrinks them with coding algorithms; these reduced
 blocks are then efficiently packed together into physical blocks. Thin
-provisioning manages the mapping from LBAs presented by VDO to where the data
-has actually been stored, and also eliminates any blocks of all zeroes.
+provisioning manages the mapping from logical block addresses presented by VDO
+to where the data has actually been stored, and also eliminates any blocks of
+all zeroes.
 
 With deduplication, instead of writing the same data more than once each
 duplicate block is detected and recorded as a reference to the original
-block. VDO maintains a mapping from logical block addresses (used by the
-storage layer above VDO) to physical block addresses (used by the storage layer
-under VDO). After deduplication, multiple logical block addresses may be mapped
+block. VDO maintains a mapping from logical block addresses (presented to the
+storage layer above VDO) to physical block addresses on the storage layer
+under VDO. After deduplication, multiple logical block addresses may be mapped
 to the same physical block address; these are called shared blocks and are
-reference-counted by the software.
+reference counted by the software.
```

Comment:
The hyphen in "reference-counted" has been removed, but is needed for consistency with the other places it is referenced.